playbooks/files/grafana/ldap.toml.j2

```diff
@@ -1,24 +1,24 @@
 # {{ ansible_managed }}
 
 [[servers]]
```

Comment:
This PR made me realise another 2 issues: inconsistent quotes (not a big deal), and the need to properly represent values in TOML. Adding quotes manually is fragile, so I think it might make sense to build a dict with all the config and pass it through the filter

```suggestion
{{
  {
    'servers': [
      {
        'host': grafana_ldap__servers__host,
        'port': grafana_ldap__servers__port,
        'use_ssl': grafana_ldap__servers__use_ssl,
        'start_tls': grafana_ldap__servers__start_tls,
        'ssl_skip_verify': grafana_ldap__servers__ssl_skip_verify,

        'bind_dn': grafana_ldap__servers__bind_dn,
        'bind_password': grafana_ldap__servers__password,

        'search_filter': grafana_ldap__servers__search_filter,
        'search_base_dns': grafana_ldap__servers__base_dns,

        'attributes': {
          'name': grafana_ldap__servers_atributes__name,
          'surname': grafana_ldap__servers_atributes__surname,
          'username': grafana_ldap__servers_atributes__username,
          'member_of': grafana_ldap__servers_atributes__member_of,
          'email': grafana_ldap__servers_atributes__email,
        },

        'group_mappings': grafana_ldap__servers_group_mappings,
      }
    ],
  } | tina_pm.common.to_toml
}}
```